src/jobflow_remote/jobs/jobcontroller.py

```diff
@@ -4105,6 +4113,163 @@ def delete_jobs(
             max_limit=max_limit,
         )
 
+    def backup_dump(
```

Comment:
This is problematic, since making the backup of all the additional stores of the JobStore might not be trivial. Since the queue part is the only one important for the update feature I would not implement this here.

Reply:
I agree, let's keep it like that.